frontend/src/Main.js

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import { BrowserRouter as Switch, Route } from 'react-router-dom';
```

Comment:
BrowserRouter doesn't work exactly the same as Switch but it works for now. Eventually we'll go back and refactor these to make sure it works correctly.